Abdul/Leaf_Similar_Trees.py

```diff
@@ -0,0 +1,33 @@
+# abdul sheikh
+# may 22, 2022
+# solution
+
+# Definition for a binary tree node.
+# class TreeNode:
+#     def __init__(self, val=0, left=None, right=None):
+#         self.val = val
+#         self.left = left
+#         self.right = right
+class Solution:
+    
+    def findLeafSum(self,root, leaf_sum):
+        if not root:
+            return 
```

Comment:
Again here you are not returning anything and your function in line 24 is returning `leaf_sum`.